docs/asdf/developer_versioning.rst

```diff
@@ -169,14 +169,14 @@ Update the asdf-standard submodule commit pointer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The asdf-standard repository is integrated into the asdf repository
-as a submodule.  To pull in new commits from the remote master (
+as a submodule.  To pull in new commits from the remote main (
```

Comment:
Can you open a matching PR in `asdf-standard`? we can then merge them right after one another and update the branch names after that.

Reply:
Sounds good.
https://github.com/asdf-format/asdf-standard/pull/371